### ISSUE TITLE
Delay

### ISSUE THREAD

**Issue by notIkibo** [v4lpt/YXN#2]
1. Give a delay for the reveal
2. The delay after reveal should be ridiculously stupid long like 10 minutes. 5 seconds doesn't hold anyone back. You don't use the app that often even if you're indecisive. And a long delay to come to accept the decision is necessary for  the app. 

Your app is more worth than 5 seconds of decision power.

**Comment by v4lpt** [comment]
You are absolutely right. I will extend the delay. I'm not entirely sure about the 10 minutes - maybe people want to ask 2 or 3 independent questions - or follow-up questions. What do you think about making the delay definitely longer, but varying the length of the delay randomly?

And regarding 1., since you obviously use the app, I think that's definitely a good idea. I'll try it out in the next release. What did you have in mind (number of seconds) for the pre-reveal delay? (A countdown would strike me as a bit too playful for the seriousness of this app.) 
:)
